sql-mode.md

```diff
@@ -7,48 +7,48 @@ aliases: ['/docs-cn/dev/sql-mode/','/docs-cn/dev/reference/sql/sql-mode/']
 
 TiDB 服务器采用不同 SQL 模式来操作，且不同客户端可以应用不同模式。SQL 模式定义 TiDB 支持哪些 SQL 语法及执行哪种数据验证检查。
 
-TiDB 启动之后采用 `SET [ SESSION | GLOBAL ] sql_mode='modes'` 设置 SQL 模式。设置 GLOBAL 级别的 SQL 模式时用户需要有 SUPER 权限，并且只会影响到从设置 SQL 模式开始后续新建立的连接（注：老连接不受影响）。SESSION 级别的 SQL 模式的变化只会影响当前的客户端。
+TiDB 启动之后采用 `SET [ SESSION | GLOBAL ] sql_mode='modes'` 设置 SQL 模式。设置 `GLOBAL` 级别的 SQL 模式时用户需要有 `SUPER` 权限，并且只会影响到从设置 SQL 模式开始后续新建立的连接（注：老连接不受影响）。`SESSION` 级别的 SQL 模式的变化只会影响当前的客户端。
```

Comment:
```suggestion
TiDB 启动之后，你可以使用 `SET [ SESSION | GLOBAL ] sql_mode='modes'` 语句设置 SQL 模式。

- 设置 `GLOBAL` 级别的 SQL 模式时用户需要有 `SUPER` 权限，并且只会影响到从设置 SQL 模式开始后续新建立的连接（注：老连接不受影响）。
- 设置 `SESSION` 级别的 SQL 模式的变化只会影响当前的客户端。

```

Reply:
Sorry， should I rechange my local files and push it again？ This is my first time to create pr. I do not know it exactly.